api/wiki_versions/serializers.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import (
+    JSONAPISerializer,
+    IDField,
+    TypeField,
+    LinksField,
+    RelationshipField,
+    VersionedDateTimeField,
+)
+
+from api.base.utils import absolute_reverse
+
+
+class WikiVersionSerializer(JSONAPISerializer):
+    id = IDField(source='_id', read_only=True)
+    type = TypeField()
+    identifier = ser.IntegerField()
+    date_modified = VersionedDateTimeField(source='date')
```

Comment:
Drive-by: this should be date_created to be compatible with the FileVersionsSerializer. Ideally, from the API's perspective, this should be identical serialization, or identical with additional information, to the FileVersionSerializer. (The thinking when making versions is that you can only create them, never update them, because updating makes a new version, so it only has a date_created).

Reply:
ah, I see, thanks!